### PR TITLE
9372: Remove Risk Level for text partial days since app installation

### PR DESCRIPTION
**Pull request by flxschmidt**
## Description
Remove Risk Level for days since app installation, regarding screen text logic

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9372

## Screenshots
Screen Shot 2022-07-07 at 08 04 48

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ENSetting/Cells/DaysSinceInstallTableViewCell.swift [hunk 1]
```diff
@@ -48,7 +48,7 @@ class DaysSinceInstallTableViewCell: UITableViewCell {
 			self.heightDidChange?()
 		}
 		
-		if viewModel.daysSinceInstall < 14 {
+		if viewModel.daysSinceInstall < 11 {
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/ENSetting/Cells/DaysSinceInstallTableViewCell.swift:51]: Should it be 10?

Comment (flxschmidt) [reply src/xcode/ENA/ENA/Source/Scenes/ENSetting/Cells/DaysSinceInstallTableViewCell.swift:51]: okay current discussion: threshold value for exposure logging = 14 days, exposure detection = 10 days. the bug description seems to be wrong. when the bug will be consistent, i will adapt the code as well again

Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/ENSetting/Cells/DaysSinceInstallTableViewCell.swift]: Can you please update the PR description?

Comment (flxschmidt) [reply src/xcode/ENA/ENA/Source/Scenes/ENSetting/Cells/DaysSinceInstallTableViewCell.swift]: done

**Review by naveeddotio — APPROVED** [review]

**Review by 30mar — APPROVED** [review]

**Pull request merged**